sql/atxs/atxs_test.go

```diff
@@ -414,6 +414,53 @@ func TestGetIDsByEpoch(t *testing.T) {
 	require.EqualValues(t, []types.ATXID{atx4.ID()}, ids3)
 }
 
+func TestGetIDsByEpochCached(t *testing.T) {
+	db := sql.InMemory(sql.WithQueryCache(true))
```

Comment:
I feel we need to update this test to assert that the fetched values were actually cached and not queried from the DB. Without `sql.WithQueryCache(true)` the test still passes.

Reply:
Fixed by adding query counter to the `Database` and checking it in the test